fields/types/localfiles/LocalFilesType.js

```diff
@@ -170,7 +171,7 @@ localfiles.prototype.addToSchema = function() {
 		delete: function(element_id) {
 			if (exists(this, element_id)) {
 				var values = this.get(field.path);
-				var value = values.find(function (val) { return val._id === element_id; });
+				var value = values.find(function (val) { return val._id == element_id; });
```

Comment:
This is a common trap with mongoose. @Rmannn @ericelliott what do you think of wrapping both values in `String(id)` and using `===`? That way the conversion is explicit rather than implicit. Otherwise we should at least have a comment explaining the intentional use.